config/opensearch_dashboards.yml

```diff
@@ -175,6 +175,7 @@
 #     darkModeUrl: ""
 #   faviconUrl: ""
 #   applicationTitle: ""
+#   useExpandedMenu: false
```

Comment:
perhaps enable? or if we intend to get this in a minor change then disableExpandedHeader?

Reply:
updated in latest revision to make so that the expanded menu remains the default.